lib/std/os/bits.zig

```diff
@@ -11,7 +11,8 @@ pub usingnamespace switch (builtin.os) {
     .netbsd => @import("bits/netbsd.zig"),
     .wasi => @import("bits/wasi.zig"),
     .windows => @import("bits/windows.zig"),
-    else => struct {},
+    else => if (@hasDecl(@import("root"), "os") and @hasDecl(@import("root").os, "bits")) @import("root").os.bits
```

Comment:
This seems sort of weird to me.

Reply:
How so?

Reply:
like.... if you wanted to import `os.bits` into your `os` struct, you would just do it yourself

Reply:
That's not what this does - this doesn't import std.os.bits into the custom os struct, it imports the custom os struct's bits struct instead of e.g. bits/windows.zig

Reply:
Isn't the main use of `os/bits.zig` to just be used in `os.zig` as: `
pub usingnamespace @import("os/bits.zig");`?

Reply:
Yeah, but if there's a reason to keep e.g. bits/linux.zig separate from linux.zig itself, then that reasoning applies equally to custom OS layers.